cli.js

```diff
@@ -10,7 +10,7 @@ import { createServer } from 'vite'
 import { MochaProtocolPlayer } from './mochaProtocolPlayer.js';
 import { MochaProtocolReporter } from './mochaProtocolReporter.js';
 
-const options = getopt({
+let options = getopt({
```

Comment:
I think it can stay "const", since you are not reassigning the options variable itself, only the keys. `const` only applies to top level.